js/mixins/dom_iterable.ts

```diff
@@ -24,19 +24,19 @@ export function DomIterableMixin<K, V, TBase extends Constructor>(
   // tslint:disable-next-line:variable-name
   const DomIterable = class extends Base {
     *entries(): IterableIterator<[K, V]> {
-      for (const entry of (this as any)[dataSymbol].entries()) {
+      for (const entry of (this as any)[dataSymbol]) {
```

Comment:
This change is also in #1056.  Maybe the best thing to do is create a seperate change that includes this that both #1056 and this one can base off of.

Reply:
I don't think need to create a seperate change in here. `for const entry of Array<k, v>` as same as `for const entry of Map<k, v>`.